Update azure-security-disk-encryption.md

DevTiw opened the pull request:
Added documentation updates to the new supported encryption scenarios enabled like LVM support for RHEL LVM 7.3 gallery image OS and data disks, update encryption settings for premium storage VMs, Azure backup and restore support for no-KEK (key encryption key) encrypted VMs etc..

azurecla commented:
Hi __@VeljkoVasic__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Please change the link syntax to the following. Thanks.

[Pricing tier recommendations](sql-database-pricing-tier-recommendations.md)

VeljkoVasic commented:
Thanks for the guidance. I commited the change. Should I create new pull request or this one will pick it up?

DevTiw committed:
Update azure-security-disk-encryption.md

Added documentation updates to the new supported encryption scenarios enabled like LVM support for RHEL LVM 7.3 gallery image OS and data disks, update encryption settings for premium storage VMs, Azure backup and restore support for no-KEK (key encryption key) encrypted VMs etc..
